docs/_news/news07.md

```diff
@@ -20,13 +20,15 @@ Curious? [Learn more](remoteoverview.html)!
 
 #### New Features and Highlights for 0.7.0
 
+**Monitoring Updates**
+- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Nodejs, Liberty and Spring projects. 
```

Comment:
- "Node.js" needs the period after the "e."
- We use the Oxford comma. Please add a comma after "Liberty." ;)